src/base/static/state/ducks/user.js

```diff
@@ -0,0 +1,69 @@
+// Selectors:
+export const userSelector = state => {
+  return state.user;
+};
+export const hasGroupAbilityInAnyDataset = ({
+  state,
+  ability,
+  submissionSet,
+  datasetSlugs,
+}) =>
+  state.user.groups
+    // Limit to datasets used on this flavor only.
+    .filter(group => datasetSlugs.includes(group.dataset_slug))
+    .some(group =>
+      group.permissions.some(
+        perm =>
+          (perm.submission_set === "*" ||
+            perm.submission_set === submissionSet) &&
+          perm.abilities.includes(ability),
+      ),
+    );
+export const hasGroupAbilityInDataset = ({
```

Comment:
Why do we have two utility methods that are doing the same thing? Instead of calling `hasGroupAbilityInDataset`, we can call `hasGroupAbilityInAnyDataset` while passing an array with a single datasetSlug for `datasetSlugs`. Then we can delete this `hasGroupAbilityInDataset` function.

I would also suggest renaming `hasGroupAbilityInAnyDataset` to `hasGroupAbilityInDatasets`.